is_prime.py

```diff
@@ -0,0 +1,13 @@
+def is_prime(a):
+    i = 2
```

Comment:
Why?

Reply:
ififi

Reply:
Please add a comment to explain this

Reply:
Please answer to my comments so that I see that you've changed something